### ISSUE TITLE
Typescript types are not yet added

### ISSUE THREAD

**Issue by landsman** [leoek/fetch-to-curl#21]
import fetchToCurl from 'fetch-to-curl';

```
TS7016: Could not find a declaration file for module 'fetch-to-curl'. '/home/landsman/projects/xy/node_modules/fetch-to-curl/lib/bundle.js' implicitly has an 'any' type.   Try `npm install @types/fetch-to-curl` if it exists or add a new declaration (.d.ts) file containing `declare module 'fetch-to-curl';`
```

**Comment by leoek** [comment]
@landsman the import is working correctly. This is just a warning that there are no typescript definitions yet. I am currently not using the library in any typescript projects and therefore can't justify the time to add these right now (even if this is probably done with little effort).

I would be very happy if you could add a file with type definitions. :)

**Comment by plaa** [comment]
As a workaround, place the following in e.g. `missing-lib-types.d.ts` in the source root of the project using fetch-to-curl:

```
declare module 'fetch-to-curl' {
  export default function fetchToCurl(...args: any[]): string;
}
```

The definitions in the npm package would probably be pretty similar, but not exactly sure what/how.

**Comment by leoek** [comment]
Next Release v0.5.0+ will include typescript types.

see commits: 81d06b3d9fe74c9663dfccaa7cde645d8fb5080c & 5dd966a68657aa99f0cab04df81655b07c52cd5d

**Comment by leoek** [comment]
released in version [0.5.0](https://github.com/leoek/fetch-to-curl/releases/tag/0.5.0)
